### DISCUSSION TITLE
Gamercards

### DISCUSSION THREAD

**Discussion by luchaos** [RetroAchievements/RAWeb#670] [Ideas]
Something like that https://cards.completionist.me/
PHP library is ready to be pulled in via composer, has plenty of customization options: https://github.com/completionist/vanity

**Comment by kdecks** [comment]
Where would they be visible? On Discord? Player page?

**Comment by luchaos** [comment]
i guess there would be a dedicated user page for that or displayed on the settings page. with bbcode, html, direct image url snippets. to be used in other forums mainly - but maybe we can find fitting spots on the site as well.
